### DISCUSSION TITLE
Where is REPO/OUT/{VERSION}.jar ?

### DISCUSSION THREAD

**Discussion by octavflorescu** [microsoft/onnxruntime-extensions#492] [Q&A] [answered]
copy JAR package from REPO/OUT/{VERSION}.jar into app/libs folder.
build and run this java project.

from the demo4j tutorial...

**Comment by wenbingl** [comment, marked as answer]
check [here](https://github.com/microsoft/onnxruntime-extensions/blob/main/java/README.md) for more information.
